src/app/core/csp.service.ts

```diff
@@ -0,0 +1,27 @@
+import { DOCUMENT } from '@angular/common';
+import { Inject, Injectable } from '@angular/core';
+import { environment } from '../../environments/environment';
+
+
+@Injectable()
+export class CspService {
+  constructor(
+    @Inject(DOCUMENT) private doc: Document
+  ) { }
+
+  register() {
```

Comment:
I think it may be worthwhile to add a comment as to what this worker is doing and what are the repercussions, e.g. if we decide to use 3rd party fonts (Google), we'll need to modify the font-src directive.

Reply:
Good point. Added URLs to info about CSP. If we ever add custom fonts or anything like that, the errors in the console will point to CSP right away. They are very descriptive.